nmostesting/Config.py

```diff
@@ -33,10 +33,10 @@
 # Number of seconds to wait after browsing for a DNS-SD advert before checking the results
 DNS_SD_BROWSE_TIMEOUT = 2
 
-# Number of seconds expected between heartbeats
+# Number of seconds expected between heartbeats for an IS-04 registry
```

Comment:
Minor quibble: There are two ways this value could be used, (a) testing Nodes conform to this expectation in IS-04-01, (b) configuring the mock Node behaviour used to test a Registry in IS-04-02. Looks like it's only used in the former way, at the moment. Maybe "from an IS-04 node to an IS-04 registry" would clearly cover that as well?